examples/data-transfer-pg/sql/db1_setup.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE fruits (id INT, name VARCHAR(50), quantity INT);
```

Comment:
Add SQL comments too. 
```suggestion
--
-- Create table for fruits data.
CREATE TABLE fruits (id INT, name VARCHAR(50), quantity INT);
```